CHANGELOG.md

```diff
@@ -22,16 +22,24 @@
 
 ## 6.0.0 (2018-10-11)
 
-- Use the build_essential resource in the source install recipe instead of the build-essential::default recipe. This way we can use the new built-in build_essential resource in Chef 14+
+- Use the build_essential resource in the source install recipe instead
```

Comment:
@MarkGibbons I’m partial to leaving the CHANGELOG & README as soft-wrapped, not hard-wrapped. GitHub renders them well, and the history is clearer, too.

Reply:
If the md override to let that be is available I'll do that.  I have no particular desires either way.